database/migrations/2022_08_15_193748_update_timeline_eras_add_position.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateTimelineErasAddPosition extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('timeline_eras', function (Blueprint $table) {
+            $table->unsignedTinyInteger('position');
```

Comment:
If you try and create a new era on a timeline, do you get an error? I have a feeling this is missing ->nullable() ;-)